sfyra/go.mod

```diff
@@ -43,8 +43,8 @@ require (
 	k8s.io/apiextensions-apiserver v0.22.3
 	k8s.io/apimachinery v0.23.0-alpha.4
 	k8s.io/client-go v0.23.0-alpha.4
-	sigs.k8s.io/cluster-api v0.4.3
```

Comment:
we can probably now try dropping the "hold" block above which holds some versions back to preserve compatibility